### DISCUSSION TITLE
Regarding Application Data

### DISCUSSION THREAD

**Discussion by PoojaGaur1708** [DMTF/libspdm#845] [Q&A] [answered]
Hi , i have done version, capabilities and negotiate algorithm, key exchange and finish.
Can you pls tell me how to send a application data . API for it.

**Comment by jyao1** [comment, marked as answer]
Please refer to https://github.com/DMTF/spdm-emu/blob/main/library/mctp_requester_lib/mctp_send_receive.c#L28, 'mctp_send_receive_data() '
